SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -673,7 +675,20 @@ Buffer {
 		buffer = super.newCopyArgs(server, bufnum).cache;
 		Dialog.openPanel({ arg path;
 			buffer.doOnInfo_(action)
-				.allocRead(path, startFrame, numFrames, { ["/b_query", buffer.bufnum] })
+			.allocRead(path, startFrame, numFrames, { ["/b_query", buffer.bufnum] })
```

Comment:
sorry @prko I missed one! this one should be indented also - then we're good to go!

Reply:
Done, I hope I correctly understood you